packages/elements/examples/nextjs/app/sign-in/[[...sign-in]]/page.tsx

```diff
@@ -188,7 +210,9 @@ export default function SignInPage() {
             <Verification name='reset_password_email_code'>
               <H3>Verify your email</H3>
 
-              <P>Please check your email for a verification code...</P>
+              <P className='text-sm'>
+                Please check your <SafeIdentifier /> for a verification code...
```

Comment:
nit: Should this be something like:
```suggestion
                We've sent a verification code to <SafeIdentifier />.
```